OpenSim/Common/Object.cpp

```diff
@@ -1367,9 +1367,15 @@ setAllPropertiesUseDefault(bool aUseDefault)
 bool Object::
 print(const string &aFileName) const
 {
-    try {
+    // Default to strict exception to avoid creating bad files
+    // but for debugging allow users to be more lenient.
```

Comment:
Could you document this in the doxygen description for `print()`?

Reply:
Done